### ISSUE TITLE
Types are not exported

### ISSUE THREAD

**Issue by mohd-akram** [meikidd/iso-639-1#68]
I get this error with the new version:

```javascript
main.js:1:21 - error TS7016: Could not find a declaration file for module 'iso-639-1'. '/private/tmp/iso-bug/node_modules/iso-639-1/src/wrapper.mjs' implicitly has an 'any' type.
  There are types at '/private/tmp/iso-bug/node_modules/iso-639-1/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'iso-639-1' library may need to update its package.json or typings.

1 import ISO6391 from "iso-639-1";
```

I believe you need to add `types`/`default` fields since the types are at a different location (`index.d.ts` while the file is at `src/wrapper.mjs`). More information [here](https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing).

To reproduce the issue:

`main.js`:

```javascript
import ISO6391 from "iso-639-1";
```

`jsconfig.json`:

```json
{
  "compilerOptions": {
    "checkJs": true,
    "target": "ES2022",
    "module": "Node16",
    "maxNodeModuleJsDepth": 0,
    "strict": true
  }
}
```

`package.json`:

```json
{
  "type": "module",
  "scripts": {
    "typecheck": "tsc -p jsconfig.json"
  },
  "dependencies": {
    "iso-639-1": "^3.0.0"
  },
  "devDependencies": {
    "typescript": "^5.1.6"
  }
}
```

Command: `npm run typecheck`

**Comment by mohd-akram** [comment]
I noticed that the ESM version is simply a wrapper. There's no need for it as Node supports importing CommonJS from ESM. Therefore, removing the entire `exports` section from `package.json` resolves the issue. The `src/wrapper.mjs` file can be removed as well.

**Comment by meikidd** [comment]
Hi @mohd-akram , thank you for the feedback. I published a new version v3.0.1 for the fix, please help to check.

**Comment by mohd-akram** [comment]
Works great, thanks @meikidd.
